config/blobs.yml

```diff
@@ -1,5 +1,5 @@
 ---
-golang/go1.5.1.linux-amd64.tar.gz:
-  object_id: 11ae530c-2e4e-40b0-9f41-43da0df71fa5
-  sha: 46eecd290d8803887dec718c691cc243f2175fe0
-  size: 77875767
+golang/go1.8.linux-amd64.tar.gz:
```

Comment:
I think it's a good idea to add a comment indicating where this blob was downloaded from.